docs/privacy-policy.md

```diff
@@ -0,0 +1,172 @@
+Privacy Policy
+==============
+
+**DRAFT**
+
+Your personal privacy is an important factor that the Sempervirens project and Numfocus 
+(our legal entity) considers in the development of each of our products and services. 
+We are committed to NOT storing ANY personal information through Sempervirens other 
+than IP addresses. We are committed to being transparent and open. This Privacy Policy 
+explains generically how we receive information from you, and what we do with that 
+information once we have it.
+
+What do we mean by "personal information?"
+------------------------------------------
+
+The phrase "personal information" means information which identifies you, such as your 
+name, email address, twitter account names, or other user names. Any information that 
+falls outside of this is "non-personal information."
+
+Combining non-personal information with personal information we consider to be 
+"personal information." Such combinations will not be publically shared. Removing all 
+personal information creates a data set that is deemed "non-personal information" 
+and may be collected.
+
+In some jurisdictions around the world, IP addresses and other similar pieces of
+information are considered personal information, though they are not "uniquely 
+identifying personal information." In an effort apply as broadly as possible, 
+Sempervirens considers IP addresses and other "non-uniquely identifying personal 
+information" to be personal information. 
+
+IP addresses are the only form of personal information that Sempervirens will collect
```

Comment:
I think this is overly simplistic. We have to address the other types of information that might potentially be personally identifying and how we will handle those. For example, who makes the decisions about if something is PII? These types of questions are super subtle and will require a very clear institutional policy (at the level of NumFOCUS).